mne/encoding/model.py

```diff
@@ -1,28 +1,66 @@
 import numpy as np
 from sklearn.base import BaseEstimator
 from scipy import linalg
+from .feature import EventsBinarizer, DataDelayer
 from ..externals.six import string_types
+from ..utils import warn
+from sklearn.base import is_regressor
+
+
+class EventRelatedRegressor(object):
+    def __init__(self, raw, events, est=None, event_id=None, tmin=-.5, tmax=.5,
```

Comment:
I was thinking about it like a regular ERP, where you'd generally plot it w/ the baseline. This object in general is still very WIP. I wanted to add it here to see what you guys thought about building on top of the EncodingModel code w/ higher-level classes that assumed MNE objects etc.